RELEASE.md

```diff
@@ -30,9 +31,23 @@
 
 Releasing a new version of PyTorch generally entails 3 major steps:
 
+0. Cutting a release branch preparations
 1. Cutting a release branch and making release branch specific changes
 2. Drafting RCs (Release Candidates), and merging cherry picks
-3. Promoting RCs to stable
+3. Promoting RCs to stable and performing release day tasks
+
+## Cutting a release branch preparations
+
+Following Requirements needs to be met in order to perform successful RC Cut:
+
+* Resolve all outstanding issues in the [milestones](https://github.com/pytorch/pytorch/milestone/28) before first RC cut is completed. After RC cut is completed follwong script should be executed from builder repo in order to validate the presence of the fixes in the release branch :
```

Comment:
Typo. Also, reference 1.11.0 milestone as an example, rather then the one that always will be used
```suggestion
* Resolve all outstanding issues in the milestones(for example [1.11.0](https://github.com/pytorch/pytorch/milestone/28))before first RC cut is completed. After RC cut is completed following script should be executed from builder repo in order to validate the presence of the fixes in the release branch :
```